tests/contexts/index.js

```diff
@@ -1,16 +1,20 @@
 const credentialsContext = require('credentials-context');
+const didContext = require('did-context');
+
 const invalidContexts = {
   veresOne: {
     url: 'https://w3id.org/veres-one/v1',
     value: require('./veresOnev1.json')
   },
   did: {
     url: 'https://w3id.org/did/v0.11',
-    value: require('./didv0.json')
```

Comment:
I'm not seeing where this `didv0.json` file is being removed from the repo.  That file should no longer be included in the vc-js repo.